libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/dashboardTigerWithCharts.spec.ts

```diff
@@ -28,3 +28,19 @@ describe("Dashboard with charts", { tags: ["pre-merge_isolated_tiger"] }, () =>
         });
     });
 });
+
+describe("Dashboard with pyramid and funnel charts", { tags: ["checklist_integrated_tiger"] }, () => {
+    beforeEach(() => {
+        Navigation.visit("dashboard/dashboard-tiger-charts");
+    });
+
+    it("should render funnel and pyramid charts correctly", () => {
```

Comment:
should rename to "should render default color legend of funnel and pyramid chart correctly"